src/components/TopBar.vue

```diff
@@ -48,80 +67,81 @@ export default {
     return {
       searchQuery: '',
       mobileSearchOpen: false,
+      profileQuery: USER_MENU_PROFILE,
     };
   },
 };
 </script>
 
 <style>
-  .top-bar__bar {
-    background-color: var(--white);
-    border-bottom: 1px solid var(--gray-30);
-    display: flex;
-    align-items: center;
-    margin-bottom: 2em;
-    position: fixed;
-    width: 100%;
-    top: 0;
-    padding: .25em;
-    z-index: var(--layerTopBar);
-  }
-    .top-bar__search-toggle {
-      border: 0;
-      background-color: transparent;
-      padding: 1em;
-      line-height: 0;
-    }
-    .top-bar__search-toggle[aria-expanded="true"] {
-      background-color: var(--gray-20);
-    }
-    .top-bar__link {
-      padding: 1.25em;
-      text-transform: uppercase;
-      font-size: .9em;
-      color: var(--black);
-      font-weight: 700;
-      text-decoration: none;
-    }
-    .top-bar__link img {
-      vertical-align: middle;
-    }
-    .top-bar__link--current  {
-      position: relative;
-    }
-    .top-bar__link--current::after {
-      content: '';
-      height: 2px;
-      background: black;
-      width: 100%;
-      position: absolute;
-      bottom: -.5em;
-      left: 0;
-    }
-    .top-bar__link--logo {
-      margin-right: auto;
-      margin-left: 0;
-      padding: 1em;
-      max-width: none; /* don't shrink on small screens */
-    }
-    .top-bar__link--logo img {
-      vertical-align: middle;
-    }
-    .top-bar .show-more {
-      padding: .5em 1em;
-    }
-    .top-bar__user-menu-toggle {
-      border: 0;
-      background-color: none;
-      padding: 0;
-    }
-      .top-bar__user-menu-toggle img {
-        margin-right: 0;
-        border-radius: var(--imageRadius);
-      }
-      .top-bar__user-menu-toggle .show-more__button-text {
-        position: absolute;
-        left: -9999em;
-        top: -9999em;
-      }
+.top-bar__bar {
```

Comment:
ok ok 😄 If you prefer no indentation, let's change it across the board. I can do that sometime in next few weeks.

Reply:
Sorry, for that.

If you have a strong feeling about that I can try to teach my editor to accept this :P